extension/src/documents.ts

```diff
@@ -5,9 +5,37 @@ import { AutoLispExt } from './extension';
 import { ProjectTreeProvider  } from "./project/projectTree";
 
 
+enum Origins {
+	OPENED = 'O',
```

Comment:
These can go back to numbered enums now. I only added the strings to visually see which enum was getting added and purged during my tests. IE, I don't think we need these to have meaningful representations in the long term because they aren't exported.